### PR TITLE
update Docker images to support ES6.0 alpha, removed groovy scripting

### PR DESCRIPTION
**Pull request by p365labs**
This commit will allow to start ES 6.0 properly (mapper-attachments plugin, scripts.inline, groovy scripts has been removed, install ingest-mapper)

### CODE REVIEW

**Comment by ruflin** [comment]
Seems like most breaking tests are type related as expected :-)

File: Makefile [hunk 1]
```diff
@@ -60,7 +60,9 @@ tests:
 	make elastica-image
 	make start
 	mkdir -p build
-	docker run --network=elastica_esnet elastica_elastica  make phpunit
+	#docker run --name=nginx -e "ES_HOST=elasticsearch" --network=elastica_esnet elastica_nginx nginx
```
Comment (ruflin) [inline Makefile]: Where is this coming from?

File: docker-compose.yml [hunk 2]
```diff
@@ -34,19 +35,19 @@ services:
     volumes:
       - esdata1:/usr/share/elasticsearch/data
     ports:
-      - 9200:9200
+      - 19200:19200
```
Comment (ruflin) [inline docker-compose.yml]: Anything wrong with using 9200?

**Comment by ruflin** [comment]
Going to merge this as the two comments above can be still discussed / fixed in a follow up PR.

**Comment by ruflin** [comment]
@p365labs Obviously thinking of this 10s after I merged, but we should update the CHANGELOG for this.

File: docker-compose.yml (hunk 2 above)
Comment (p365labs) [inline docker-compose.yml]: no my fault :) something that should be removed

**Pull request merged**